src/private/notifications.ts

```diff
@@ -0,0 +1,34 @@
+import { ensureInitialized, sendMessageRequestToParent } from '../internal/internalAPIs';
+import { GlobalVars } from '../internal/globalVars';
+import { TrouterNotificationPayload } from '../public/interfaces';
+
+/**
+ * Namespace to interact with the notifications to-from the SDK.
+ * The notification payload received is passed down to the app.
+ *
+ * @private
+ * Hide from docs
+ */
+export namespace notifications {
+  GlobalVars.handlers['trouter.notificationRecieved'] = handleProcessNotifications;
+
+  function handleProcessNotifications(payload: TrouterNotificationPayload): void {
+    if (GlobalVars.notificationHandler) {
```

Comment:
we are registering these kind of handlers inside internaleAPIs,ts 
![image](https://user-images.githubusercontent.com/1689428/82107716-b0b85d00-96de-11ea-9ca5-e7f30f0f8bd3.png)

Lets follow consistent approach